Address warning when running `yarn test` in @ez/vue

There's a warning that shows up when running `yarn test:vue`. The onResize not being passed, while it should be optional.
```
Console
@ez/vue:     console.error ../../node_modules/vue/dist/vue.runtime.common.dev.js:571
@ez/vue:       [Vue warn]: Invalid handler for event "resize": got undefined
@ez/vue:       
@ez/vue:       found in
@ez/vue:       
@ez/vue:       ---> <Chart>
@ez/vue:              <BarChart>
@ez/vue:                <Root>
```

@marrouchi I created a bug, but it seems like it's not the case 🤔 
I've already installed the `@vue/cli` globally, and the issue still exists 🤔 seems like there is missing more... 

after `npm i -g @vue/cli` we have:
```bash
> eazychart-vue@0.1.16-alpha.0 test
> vue-cli-service test:unit

sh: vue-cli-service: command not found
``` 


Then, after `npm i -g @vue/cli-service` when running `npm run test` have:

```bash

> eazychart-vue@0.1.16-alpha.0 test
> vue-cli-service test:unit

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '@vue/cli-plugin-babel'
Require stack:
```

and then after `npm i -g @vue/cli-plugin-babel`, when running `npm run test` another one...

```bash

> eazychart-vue@0.1.16-alpha.0 test
> vue-cli-service test:unit

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '@vue/cli-plugin-eslint'
Require stack:
```


I feel like it's never going to end 😅  there must be another way 😖

When running `npm install` in the `packages/ez-vue` directory, why there's this error?
```bash
npm WARN ERESOLVE overriding peer dependency
npm WARN ERESOLVE overriding peer dependency
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/eazychart-dev - Not found
npm ERR! 404
npm ERR! 404  'eazychart-dev@^0.1.15-alpha.0' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

--- a/packages/ez-vue/tests/unit/recipes/area/AreaChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/area/AreaChart.spec.tsx
@@ -11,6 +11,7 @@ describe('AreaChart', () => {
   it('renders an area chart', async () => {
     const wrapper = render(AreaChart, {
       propsData: {
+        onResize: () => undefined,
         data: pointsData as unknown as RawData,
         area: {
           stroke: 'red',

--- a/packages/ez-vue/tests/unit/recipes/bar/BarChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/bar/BarChart.spec.tsx
@@ -10,6 +10,7 @@ describe('BarChart', () => {
   it('renders a bar chart', async () => {
     const wrapper = render(BarChart, {
       propsData: {
+        onResize: () => undefined,
         data: rawData,
         colors,
         grid: { directions: [] },

--- a/packages/ez-vue/tests/unit/recipes/column/ColumnChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/column/ColumnChart.spec.tsx
@@ -10,6 +10,7 @@ describe('ColumnChart', () => {
   it('renders a column chart', async () => {
     const wrapper = render(ColumnChart, {
       propsData: {
+        onResize: () => undefined,
         data: rawData,
         colors,
         grid: { directions: [] },

--- a/packages/ez-vue/tests/unit/recipes/column/LineColumnChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/column/LineColumnChart.spec.tsx
@@ -10,6 +10,7 @@ describe('LineColumnChart', () => {
   it('renders a line & column chart', async () => {
     const wrapper = render(LineColumnChart, {
       propsData: {
+        onResize: () => undefined,
         data: rawData,
         colors,
         grid: { directions: [] },

--- a/packages/ez-vue/tests/unit/recipes/line/LineChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/line/LineChart.spec.tsx
@@ -11,6 +11,7 @@ describe('LineChart', () => {
   it('renders a line chart', async () => {
     const wrapper = render(LineChart, {
       propsData: {
+        onResize: () => undefined,
         data: pointsData as unknown as RawData,
         line: {
           stroke: 'red',

--- a/packages/ez-vue/tests/unit/recipes/line/LineErrorMarginChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/line/LineErrorMarginChart.spec.tsx
@@ -10,6 +10,7 @@ describe('LineErrorMarginChart', () => {
   it('renders a line error margin chart', async () => {
     const wrapper = render(LineErrorMarginChart, {
       propsData: {
+        onResize: () => undefined,
         data: pointsWithMarginData,
         line: {
           stroke: 'red',

--- a/packages/ez-vue/tests/unit/recipes/pie/IrregularPieChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/pie/IrregularPieChart.spec.tsx
@@ -8,12 +8,15 @@ import 'tests/mocks/ResizeObserver';
 
 describe('IrregularPieChart', () => {
   it('renders a irregular pie chart', async () => {
+    const propsData = {
+      onResize: () => undefined,
+      data: rawData,
+      colors,
+      dimensions,
+    };
+
     const wrapper = render(IrregularPieChart, {
-      propsData: {
-        data: rawData,
-        colors,
-        dimensions,
-      },
+      propsData,
     });
 
     await Vue.nextTick();
@@ -21,11 +24,7 @@ describe('IrregularPieChart', () => {
     expect(wrapper.container.innerHTML).toMatchSnapshot();
 
     const wrapper2 = render(IrregularPieChart, {
-      props: {
-        data: rawData,
-        colors,
-        dimensions,
-      },
+      props: propsData,
     });
 
     await wrapper2.updateProps({

--- a/packages/ez-vue/tests/unit/recipes/pie/PieChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/pie/PieChart.spec.tsx
@@ -8,12 +8,15 @@ import 'tests/mocks/ResizeObserver';
 
 describe('PieChart', () => {
   it('renders a pie chart', async () => {
+    const propsData = {
+      onResize: () => undefined,
+      data: rawData,
+      colors,
+      dimensions,
+    };
+
     const wrapper = render(PieChart, {
-      propsData: {
-        data: rawData,
-        colors,
-        dimensions,
-      },
+      propsData,
     });
 
     await Vue.nextTick();
@@ -21,11 +24,7 @@ describe('PieChart', () => {
     expect(wrapper.container.innerHTML).toMatchSnapshot();
 
     const wrapper2 = render(PieChart, {
-      props: {
-        data: rawData,
-        colors,
-        dimensions,
-      },
+      props: propsData,
     });
 
     await wrapper2.updateProps({

--- a/packages/ez-vue/tests/unit/recipes/pie/RadialChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/pie/RadialChart.spec.tsx
@@ -8,12 +8,15 @@ import 'tests/mocks/ResizeObserver';
 
 describe('RadialChart', () => {
   it('renders a Radial chart', async () => {
+    const propsData = {
+      onResize: () => undefined,
+      data: rawData,
+      colors,
+      dimensions,
+    };
+
     const wrapper = render(RadialChart, {
-      propsData: {
-        data: rawData,
-        colors,
-        dimensions,
-      },
+      propsData,
     });
 
     await Vue.nextTick();
@@ -21,11 +24,7 @@ describe('RadialChart', () => {
     expect(wrapper.container.innerHTML).toMatchSnapshot();
 
     const wrapper2 = render(RadialChart, {
-      props: {
-        data: rawData,
-        colors,
-        dimensions,
-      },
+      props: propsData,
     });
 
     await wrapper2.updateProps({

--- a/packages/ez-vue/tests/unit/recipes/pie/SemiCircleChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/pie/SemiCircleChart.spec.tsx
@@ -7,12 +7,14 @@ import 'tests/mocks/ResizeObserver';
 
 describe('SemiCircleChart', () => {
   it('renders a semi-circle chart', async () => {
+    const propsData = {
+      onResize: () => undefined,
+      data: rawData,
+      colors,
+      dimensions,
+    };
     const wrapper = render(SemiCircleChart, {
-      propsData: {
-        data: rawData,
-        colors,
-        dimensions,
-      },
+      propsData,
     });
 
     await Vue.nextTick();
@@ -20,11 +22,7 @@ describe('SemiCircleChart', () => {
     expect(wrapper.container.innerHTML).toMatchSnapshot();
 
     const wrapper2 = render(SemiCircleChart, {
-      props: {
-        data: rawData,
-        colors,
-        dimensions,
-      },
+      props: propsData,
     });
 
     await wrapper2.updateProps({

--- a/packages/ez-vue/tests/unit/recipes/scatter/BubbleChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/scatter/BubbleChart.spec.tsx
@@ -7,24 +7,26 @@ import 'tests/mocks/ResizeObserver';
 
 describe('BubbleChart', () => {
   it('renders a bubble chart', async () => {
-    const wrapper = render(BubbleChart, {
-      propsData: {
-        data: rawData,
-        bubble: {
-          domainKey: 'amount',
-          minRadius: 1,
-          maxRadius: 10,
-          fill: 'blue',
-        },
-        grid: { directions: [] },
-        dimensions,
-        xAxis: {
-          domainKey: 'value',
-        },
-        yAxis: {
-          domainKey: 'amount',
-        },
+    const propsData = {
+      onResize: () => undefined,
+      data: rawData,
+      bubble: {
+        domainKey: 'amount',
+        minRadius: 1,
+        maxRadius: 10,
+        fill: 'blue',
       },
+      grid: { directions: [] },
+      dimensions,
+      xAxis: {
+        domainKey: 'value',
+      },
+      yAxis: {
+        domainKey: 'amount',
+      },
+    };
+    const wrapper = render(BubbleChart, {
+      propsData,
     });
 
     await Vue.nextTick();
@@ -32,23 +34,7 @@ describe('BubbleChart', () => {
     expect(wrapper.container.innerHTML).toMatchSnapshot();
 
     const wrapper2 = render(BubbleChart, {
-      propsData: {
-        data: rawData,
-        bubble: {
-          domainKey: 'amount',
-          minRadius: 1,
-          maxRadius: 10,
-          fill: 'blue',
-        },
-        grid: { directions: [] },
-        dimensions,
-        xAxis: {
-          domainKey: 'value',
-        },
-        yAxis: {
-          domainKey: 'amount',
-        },
-      },
+      propsData,
     });
 
     await wrapper2.updateProps({

--- a/packages/ez-vue/tests/unit/recipes/scatter/ScatterChart.spec.tsx
+++ b/packages/ez-vue/tests/unit/recipes/scatter/ScatterChart.spec.tsx
@@ -7,22 +7,25 @@ import 'tests/mocks/ResizeObserver';
 
 describe('ScatterChart', () => {
   it('renders a scatter chart', async () => {
-    const wrapper = render(ScatterChart, {
-      propsData: {
-        data: rawData,
-        point: {
-          radius: 6,
-          color: 'red',
-        },
-        grid: { directions: [] },
-        dimensions,
-        xAxis: {
-          domainKey: 'value',
-        },
-        yAxis: {
-          domainKey: 'amount',
-        },
+    const propsData = {
+      onResize: () => undefined,
+      data: rawData,
+      point: {
+        radius: 6,
+        color: 'red',
       },
+      grid: { directions: [] },
+      dimensions,
+      xAxis: {
+        domainKey: 'value',
+      },
+      yAxis: {
+        domainKey: 'amount',
+      },
+    };
+
+    const wrapper = render(ScatterChart, {
+      propsData,
     });
 
     await Vue.nextTick();
@@ -30,21 +33,7 @@ describe('ScatterChart', () => {
     expect(wrapper.container.innerHTML).toMatchSnapshot();
 
     const wrapper2 = render(ScatterChart, {
-      propsData: {
-        data: rawData,
-        point: {
-          radius: 6,
-          color: 'red',
-        },
-        grid: { directions: [] },
-        dimensions,
-        xAxis: {
-          domainKey: 'value',
-        },
-        yAxis: {
-          domainKey: 'amount',
-        },
-      },
+      propsData,
     });
 
     await wrapper2.updateProps({

